engine/src/configuration/configuration.h

```diff
@@ -380,6 +380,7 @@ struct PhysicsConfig {
     float max_lost_target_live_time{30.0F};
     float percent_missile_match_target_velocity{1.0F};
     double game_speed{1.0};
+    bool game_speed_lying{false};
```

Comment:
Why is this change here ?
looks unrelated to the PR

Reply:
This is used by `PrettyDistanceString`, which I refactored. I also moved it from the xml configuration (vegastrike.config) to the new config.json.

Reply:
OK, I missed that